docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc

```diff
@@ -0,0 +1,178 @@
+standard WildFly variant.
+
+The expectation is on any given release date, both standard WildFly and WildFly Preview will be released.
+
+[WARNING]
+====
+
+A WildFly Preview release will have the same version number and suffix (Beta, Final, etc) as the
+main WildFly release, but regardless of the suffix, a WildFly Preview release should be treated
+as a Technical Preview release.
+====
+
+== Getting WildFly Preview
+
+The zip or tar.gz file for WildFly Preview is available at https://wildfly.org/downloads
+right next to the main WildFly release files for the same version.
+
+For bootable jar users and Galleon CLI users, we provide a Galleon feature pack for WildFly Preview. The
+Galleon feature pack location for the feature pack is ``wildfly-preview@maven(org.jboss.universe:community-universe)``.
+This feature pack is the WildFly Preview analogue to main WildFly's ``wildfly@maven(org.jboss.universe:community-universe)``.
+
+== WildFly Preview and Jakarta EE 9
+
+The primary thing currently being showcased in WildFly Preview is support for Jakarta EE 9. Jakarta EE 9 is
+link:https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan[primarily aimed] at
+transitioning the EE APIs from the javax.* package namespace to the jakarta.* namespace. Some techologies were made
+optional in EE 9 and others were pruned from the platform, but for all the technologies retained from EE 8, the APIs
+and expected behaviors are unchanged from EE 8, except for the very significant change in the package names.
+
+This package rename is a big change that is going to take a while to percolate through the EE ecosystem, e.g. for the
+many projects that compile against the EE APIs to provide versions that use jakarta.*. While this happens the WildFly
+project wants to continue to deliver new features and fixes to our community, so the standard WildFly distribution
+continues to provide the EE 8 APIs. But WildFly Preview's remit of providing a look at coming technologies makes it a
+good fit for letting our users experiment with EE 9.
+
+WildFly Preview is a compatible implementation of the Jakarta EE 9.0 Web Profile. Certification of the full platform
+is in progress.
+
+=== EE 9 Via Bytecode Transformation and the 'wildfly-preview' Galleon Feature Pack
+
+The large majority of the libraries included in WildFly Preview that were compiled against EE APIs were based on the
+EE 8 APIs in the javax.* packages. This includes the libraries produced from WildFly's own code base.  But the EE API
+libraries available in the WildFly Preview runtime all use the jakarta.* packages. How can this work?
+
+The solution we've come up with for this is to use the new ``wildfly-preview`` Galleon feature pack to convert any
+code using the EE 8 javax.* APIs to jakarta.* _at the time that it provisions a server installation_. (Recall that any
+WildFly server installation -- including the ones we zip up and make available on https://wildfly.org/downloads -- is
```

Comment:
s/https://wildfly.org/downloads -- is/link:https://wildfly.org/downloads[https://wildfly.org/downloads] -- is

It is not well rendered. The last `-- is` is taken as part of the link. Notice the `link:` is optional if the link starts with http, but for consistency across all the document, the suggestion is to use `link:`